base/fitness_model.py

```diff
@@ -234,14 +242,35 @@ def calc_node_frequencies(self):
             for time in self.timepoints[:-1]:
                 node.delta_freqs[time] = np.asscalar(interpolation(time + self.delta_time))
```

Comment:
@huddlej what are these `delta_freqs` for? as defined, they are just freqs. I am also not too happy with this being `defaultdicts`. This will generate `node.delta_freqs[time]=0.0` for any value that you have not set and possibly masks nasty errors.